api/v1alpha1/groupversion_info.go

```diff
@@ -15,8 +15,8 @@ limitations under the License.
 */
 
 // Package v1alpha1 contains API Schema definitions for the terra v1alpha1 API group
-//+kubebuilder:object:generate=true
-//+groupName=terra.terra-rebels.org
+// +kubebuilder:object:generate=true
```

Comment:
Remove the whitespace in this comment. Afaik conventions dictate that machine generated code should not have whitespaces. You can read more here @ https://github.com/kubernetes-sigs/kubebuilder/issues/1384